soccer/src/soccer/planning/planner/plan_request.hpp

```diff
@@ -26,9 +27,8 @@ namespace planning {
 struct PlanRequest {
```

Comment:
Could you add a docstring for this struct since there was some changes in the struct members.

Reply:
Each of the params has its own mini-docstring, did you want me to move that all to the top?